web/static.go

```diff
@@ -139,3 +145,18 @@ func unsupportedBrowserScriptHandler(w http.ResponseWriter, r *http.Request) {
 	templatesDir, _ := templates.GetTemplateDirectory()
 	http.ServeFile(w, r, filepath.Join(templatesDir, "unsupported_browser.js"))
 }
+
+func getOGDescriptionMetaTag(c *Context) string {
+	siteDescription := model.TeamSettingsDefaultCustomDescriptionText
+	customSiteDescription := c.App.Srv().Config().TeamSettings.CustomDescriptionText
+	if customSiteDescription != nil && *customSiteDescription != siteDescription {
+		siteDescription = *customSiteDescription
+	}
+
+	if siteDescription == "" {
+		return ""
+	}
+
+	ogDescriptionTemplate := "<meta property=\"og:description\" content=\"%s\" />"
+	return fmt.Sprintf(ogDescriptionTemplate, siteDescription)
```

Comment:
We need to make sure we escape any double quotes that may be present in `siteDescription` before injecting it here. We can use `html.EscapeString(s)` for this

Reply:
Good catch!

Reply:
Thanks for the comment @mickmister. I tried a custom description as `"/><meta property="og:x" content="xxx`.

without EscapeString: 
```
<meta property="og:description" content=""/>
<meta property="og:x" content="xxx">
```

with EscapeString:
```
<meta property="og:description" content="&quot;/><meta property=&quot;og:x&quot; content=&quot;xxx">
```

Thanks for it! 👍🏻